### ISSUE TITLE
Uploading additional pictures in DevLogs busted

### ISSUE THREAD

**Issue by Tricky1975** [gamejolt/issue-tracker#739]
![image](https://cloud.githubusercontent.com/assets/11202073/17457488/f4cd6806-5bf9-11e6-9421-8a094de3f957.png)

This is what I got when I uploaded additional pictures to my text-devlog.
When pasting in the MarkDown tags I can get from clicking the paperclip, I just get a picture call to a broken URL.

P.S. Uploading a picture as a log entry on its own, DOES work, though.

**Comment by hworld** [comment]
I haven't been able to reproduce this. I've tried on multiple computers on site and in the Client. I've asked others to try as well. Is it still happening?

**Comment by Tricky1975** [comment]
Yup... Still no good. Now I deem it possible the picture I used was too big, but then I think a proper error message would be in order.

This was the picture I tried it with, and I used Opera for Mac
![excalibur - final 003](https://cloud.githubusercontent.com/assets/11202073/17461911/984aa39e-5c9b-11e6-96ff-798d6b66d1bc.PNG)

**Comment by hworld** [comment]
Wow, I really don't know what could be wrong... I tried uploading and it worked just fine. You can even see here: https://n3b6p5n5.ssl.hwcdn.net/data/fireside/posts/2/165/172165/media/984aa39e-5c9b-11e6-96ff-798d6b66d1bc-w9zgytmy.png

I'm lost on this one. Everyone else seems to be able to upload as well. Have you tried other images?

**Comment by Tricky1975** [comment]
Smaller pictures appear to work.
I've filmed what exactly happens to me when I upload something larger.

https://youtu.be/TYboKb5gWl4

(Note, YouTube was still processing when I posted this link, so if it doesn't show anything, you'll need to come back later).

**Comment by Tricky1975** [comment]
I've tried this with the link you gave. 
~~~MarkDown
![](https://n3b6p5n5.ssl.hwcdn.net/data/fireside/posts/2/165/172165/media/984aa39e-5c9b-11e6-96ff-798d6b66d1bc-w9zgytmy.png)
~~~
And I get this when I preview:

![image](https://cloud.githubusercontent.com/assets/11202073/17463655/e1975b3a-5cca-11e6-9afd-1c7cbd924181.png)

And when I click your link the browser forces me to download.

**Comment by hworld** [comment]
Are you able to try a different browser just for shiggles?

**Comment by Tricky1975** [comment]
Sure... I shall try this in Firefox and tell you what happens.

**Comment by YLivay** [comment]
Do the console log trick and see what it says.
This truly is a mystery, I managed to upload it in dev without a problem as well S:

**Comment by Tricky1975** [comment]
I shall try this later tonight in FireFox (as Opera does not seem to have a Console Log or at least I can't find it).

**Comment by YLivay** [comment]
So what do we do with this? is there any way to go forward? I couldn't recreate this.

**Comment by Tricky1975** [comment]
@YLivay 
Did you try to recreate that on mac (I use El Capitan) or on a different OS.
I've only tried this on Mac with multiple browsers and even the client all causing the same issue. I didn't yet have the chance to try this on another machine.

Point is that the picture above does show here on GitHub (I mean, it's hosted on GitHub).
The link David posted above doesn't work for me, but forces me to download. When I open the Download I get the picture though.Smaller images work fine for me though, but the large one don't.

BTW, I've tried that link again to see if the issue still persists, and... it does...
